catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/NitfUtilities.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.transformer.nitf;
+
+import java.nio.file.Paths;
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.util.Date;
+import java.util.Map;
+import org.apache.commons.lang.StringUtils;
+import org.codice.ddf.platform.util.properties.PropertiesLoader;
+import org.codice.imaging.nitf.core.common.DateTime;
+
+public class NitfUtilities {
+
+  private static final String FIPS_TO_ISO_3_PROPERTY_PATH =
+      Paths.get(System.getProperty("karaf.etc"), "fipsToIso.properties").toString();
+
+  private static final Map<String, String> FIPS_TO_ISO3_MAP = getFipsMap();
```

Comment:
Unfortunately we cannot get a reference to the `LocalCountryCodeConverter` service available in platform. When we create NITF attributes, we do not do it through blueprint.